client/my-sites/checkout/composite-checkout/lib/translate-cart.ts

```diff
@@ -123,9 +123,7 @@ export function createTransactionEndpointCartFromResponseCart( {
 		blog_id: siteId || '0',
 		cart_key: siteId || 'no-site',
 		create_new_blog: siteId ? false : true,
-		is_jetpack_checkout: responseCart.products.some(
-			( product ) => product.extra.isJetpackCheckout
-		),
+		is_jetpack_checkout: false,
```

Comment:
This will always be false. 

If the following code block is true,
```
responseCart.products.some(
			( product ) => product.extra.isJetpackCheckout
		),
```
Then, the runtime takes a different code path and never reaches this one (see the `if` statement right above this code block).